src/main/java/io/rebloom/client/cf/IteratorDataPair.java

```diff
@@ -0,0 +1,19 @@
+package io.rebloom.client.cf;
+
+public class IteratorDataPair {
```

Comment:
Two different things come to mind:
1. Why don't we use `Map.Entry<***String***, byte[]>` or `Map.Entry<Long, byte[]>` instead of creating a new class?
2. (If we are certain to create a new class) `IteratorDataPair` may run into confusion if a similar feature comes in future. A name like `ScanDumpPair` is a safer option.

*** Please see my comment in constructor why I wrote `String`.

Reply:
@sazzad16 The only thing that bothers me about using `Map.Entry` is that it has the word `Map` in it, implying that they are entries in a `Map` so if feels wrong semantically (whenever I run into needed real Tuples I pull the Java Tuples library, but didn't want to bring any external dependencies if possible). Let me do a quick survey of libs to see how prevalent is to use `Map.Entry` without the `Map` being involved :-)

Reply:
@gkorland ping

Reply:
I think Map.Entry should be OK, we used it in jedis xread too.